perceval/backend.py

```diff
@@ -533,6 +533,111 @@ def setup_cmd_parser(cls):
         raise NotImplementedError
 
 
+class BackendItemsGenerator:
```

Comment:
This class should be a context manager, so it should contain the required methods, `__enter__` and `__exit__`.